app/policies/carrier_policy.rb

```diff
@@ -0,0 +1,33 @@
+class CarrierPolicy < ApplicationPolicy
+  attr_reader :user, :members
+
+  def initialize(user, members)
+    @user = user
+    @members = members
```

Comment:
I believe the second argument here would be the carrier, not `members`. Please also remember to change the `attr_reader` on line 2! :-)